src/audio/channel_map.c

```diff
@@ -14,6 +14,10 @@
 #include <stdint.h>
 #include <stdlib.h>
 
+#ifdef __ZEPHYR__
+LOG_MODULE_REGISTER(channel_map, LOG_LEVEL_INF);
+#endif
```

Comment:
doesn't this mean, that after this change all `tr_*()` prints will disappear from sof-logger output? If so, it seems we aren't ready for this change yet?

Reply:
We are not fully ready yet, so this is only a PR for review. I will continue the work on pipe_xxx, comp_xxx calls. Finally, for SOF on Zephyr, all log calls will use zephyr API and Zephyr provides very flexible ways to switch logging backend.